app/src/androidTest/java/com/github/se/cyrcle/ui/list/ListScreenTest.kt

```diff
@@ -67,12 +70,14 @@ class ListScreenTest {
     mockUserRepository = MockUserRepository()
     mockParkingRepository = MockParkingRepository()
     mockImageRepository = MockImageRepository()
+    authenticator = MockAuthenticationRepository()
```

Comment:
I think that for the sake of coherence, this should be called `mockAuthenticatorRepository`, it feels a bit weird to have only this as naked as it is